common/Environments/Helpers/ComputeSystemHelpers.cs

```diff
@@ -9,6 +9,7 @@
 using System.Threading.Tasks;

```

Comment:
There's a variable `curentProperties` in this file, should be `currentProperties`

Reply:
updpated thanks